### PR TITLE
Added spanish translations responding to issue #1321

### PR DESCRIPTION
**Pull request by manueldose**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe: Write a spanish translation of pages specified in issue #1321, critical css / flash prevention / create a plugin / use netlify cms

## What is the current behavior?

Issue Number: 1321

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by SanderElias** [comment]
@manueldose We are waiting for @Villanuevand  to approve, then we will merge your work!
Thanks for the translation!

File: docs/Reference/plugins/built-in-plugins/criticalcss_es.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+published: true
+lang: es
+position: 100
+---
+
+# `critical-css` Plugin
+
+## Información general
+
+Scully usa este plugin para incorporar el CSS crítico above the fold en el HTML, y luego con **lazy-load** incorpora el CSS que se necesita para el resto de la página. Esto eliminará el bloqueo de CSS. Estará listo antes de que arranque su SPA.
```
[the remark this answers is not in the document]
Comment (Villanuevand) [reply docs/Reference/plugins/built-in-plugins/criticalcss_es.md:12]: I would like to understand why left "above the fold" in English?

File: docs/Reference/plugins/built-in-plugins/criticalcss_es.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      postRenderers: = [criticalCSS],
+    },
+     /** más configuración de rutas aquí */
+  }
+}
+```
+
+## Ajustes
+
+Puede configurar este plugin utilizando el ayudante setPluginConfig de esta manera:
```
Comment (Villanuevand) [inline docs/Reference/plugins/built-in-plugins/criticalcss_es.md:57]:
```suggestion
Puede configurar este plugin utilizando el ayudante setPluginConfig de esta manera:
```

Comment (Villanuevand) [reply docs/Reference/plugins/built-in-plugins/criticalcss_es.md:57]: Let's keep "helper" instead use "ayudante"

File: docs/learn/create-a-plugin/create-a-plugin_es.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+position: 20
+---
+
+## [Información general](#informacion-general)
+
+Scully tiene un robusto sistema de complementos que le permite modificar los datos antes, durante y después de que se genere la aplicación. Por ejemplo, una vez creadas las rutas, puede crear un mapa del sitio o un feed RSS. El sistema del plugin brinda fácil acceso a todas las rutas una vez que se han encontrado y generado. En esta guía, aprenderá sobre los distintos tipos de plugins y cómo puede crear su propio plugin personalizado.
+
+## [Tipos de Plugin](#tipos-de-plugin)
+
+Hay varios tipos de plugins que puede crear para su aplicación Scully, dependiendo de cuándo en el proceso de compilación necesite usarlo. Estos son los diferentes tipos de plugins que puede crear y una breve explicación de cada uno, [pulled directly from Scully's docs](https://scully.io/docs/Reference/plugins/overview/):
```
Comment (Villanuevand) [inline docs/learn/create-a-plugin/create-a-plugin_es.md:13]:
```suggestion
Hay varios tipos de plugins que puede crear para su aplicación Scully, dependiendo de cuándo en el proceso de compilación necesite usarlo. Estos son los diferentes tipos de plugins que puede crear y una breve explicación de cada uno, [obtenidos desde la documentacion de Scully](https://scully.io/docs/Reference/plugins/overview/):
```

**Review by Villanuevand — COMMENTED** [review]
@manueldose 
I've just added some comments. 

Great work!

**Comment by SanderElias** [comment]
@Villanuevand What's the state of this PR?

**Comment by SanderElias** [comment]
@manueldose I was about to merge this, but:
image
Can you sign the CLA, then I can merge your change in

**Review by SanderElias — APPROVED** [review]

**Pull request merged**